sdk/messaging/azeventgrid/autorest.md

```diff
@@ -65,4 +65,16 @@ directive:
       - response_types.go
     where: $
     transform: return $.replace(/\*CloudEvent/g, "messaging.CloudEvent");
+
+  # remove the 'Interface any' that's generated for an empty response object.
```

Comment:
You could also use a transform to change the definition of the 200 response to not include a schema (not sure which is easier).